sqlserver/datadog_checks/sqlserver/sqlserver.py

```diff
@@ -877,9 +877,14 @@ def collect_index_usage_metrics(self):
         if not self._index_usage_last_check_ts or now - self._index_usage_last_check_ts > interval:
             self._index_usage_last_check_ts = now
             self.log.debug('Collecting index usage statistics')
-            db_names = [d.name for d in self.databases] or [
-                self.instance.get('database', self.connection.DEFAULT_DATABASE)
-            ]
+            default_database = self.instance.get('database', self.connection.DEFAULT_DATABASE)
+            # Filter out tempdb as the query might be blocking and it's index usage information is not relevant
+            include_temdb_in_metric = is_affirmative(self.instance.get('include_index_usage_metrics_tempdb', False))
```

Comment:
it will be good if we can move this flag to `config.py`. i know we still have many other config related code sit in `sqlserver.py` but i want to start moving them to config

Reply:
```suggestion
            include_index_usage_metrics_tempdb = is_affirmative(self.instance.get('include_index_usage_metrics_tempdb', False))
```

Reply:
thx, will do